File: 프로그래머스/0/181934. 조건 문자열/조건 문자열.java
class Solution {
    public int solution(String ineq, String eq, int n, int m) {
        int result = 0;
        String op = ineq + eq;
        switch(op){
                case "<=":
                    if(n <= m) result = 1;
                    break;
                case ">=":
                    if(n >= m) result = 1;
                    break;
                case "<!":
                    if(n < m) result = 1;
                    break;
                case ">!":
                    if(n > m) result = 1;
                    break;
        }
        return result;
    }
}

File: 프로그래머스/0/181934. 조건 문자열/조건 문자열.js
function solution(ineq, eq, n, m) {
    let op = ineq + eq;
    let answer;
    if(op == ">=") answer = n >= m;
    else if(op == "<=") answer = n <= m;
    else if(op == ">!") answer = n > m;
    else answer = n < m;
    
    return Number(answer);
}